x-pack/spec/monitoring/inputs/metrics/state_event_factory_spec.rb

```diff
@@ -47,7 +47,7 @@
     LogStash::SETTINGS.set_value("monitoring.enabled", false)
   end
 
-  let(:pipeline) { LogStash::Pipeline.new(config) }
+  let(:pipeline) { LogStash::JavaPipeline.new(config, nil) }
```

Comment:
`nil` as second parameter shouldn't be needed due to JavaPipeline#initialize signature:
```ruby
def initialize(pipeline_config, namespaced_metric = nil, agent = nil)
```